posthog/api/organization.py

```diff
@@ -95,13 +97,32 @@ def create(self, validated_data: Dict, *args: Any, **kwargs: Any) -> Organizatio
         organization, _, _ = Organization.objects.bootstrap(self.context["request"].user, **validated_data)
         return organization
 
+    def update(self, organization: Organization, validated_data: Dict, *args: Any, **kwargs: Any) -> Organization:  # type: ignore
```

Comment:
Wrapping this in `@transaction.atomic` will ensure consistency for:

 1. reads that happen between the two updates
 2. cases where the second update fails e.g. if you pass in `[["uuid"]]` as the `members_to_send_plugin_alerts`

Or you could do the update as one query, setting the `send_plugin_alerts` to an expression equivalent to `user__uuid__in=self.context["request"].data["members_to_send_plugin_alerts"]` but at any rate you probably want to ensure atomicity around the whole thing.

Ensuring the field has passed through the serialized would be an improvement as well.